examples/istioctl-analyze/README.md

```diff
@@ -0,0 +1,37 @@
+# Istioctl Analyze
+
+The `istioctl-analyze` config function detects potential issues with your Istio
+configuration and output errors as results. This example invokes the
```

Comment:
1. "output errors as results" assumes the reader is aware of `results`? Rephrase
2. "config function" -> "KRM Config Function". It's a proper-noun, otherwise it's ambiguous.

Reply:
Rephrased the intro